### ISSUE TITLE
Fatal error: Cannot redeclare tap()

### ISSUE THREAD

**Issue by vmitchell85** [vitr/valet4windows#9]
I encountered an issue where I would get this error when using [jigsaw ](https://github.com/tightenco/jigsaw) by [tightenco](https://github.com/tightenco).

I fixed my issue by wrapping your `function tap()` in a `if (! function_exists('tap'))` statement.

I'm not sure what other functions would benefit from this.

[illuminate/support](https://github.com/illuminate/support) (which jigsaw uses) has done this with every function. See [helpers.php](https://github.com/illuminate/support/blob/master/helpers.php#L808)

**Comment by vitr** [comment]
yeah, this is the change they made in original project, I should update my version accordingly, thanks
